events/app.py

```diff
@@ -0,0 +1,113 @@
+import os
+import logging
+import datetime
+import pymongo
+# import pprint
+
+from flask import Flask, request, Response
+
+app = Flask(__name__)
+
+# connect to MongoDB Atlas database
+MONGODB_ATLAS_USERNAME = os.environ.get(
+    "MONGODB_ATLAS_USERNAME")
+MONGODB_ATLAS_PASSWORD = os.environ.get(
+    "MONGODB_ATLAS_PASSWORD")
+MONGODB_ATLAS_CLUSTER_ADDRESS = os.environ.get(
+    "MONGODB_ATLAS_CLUSTER_ADDRESS")
+PYMONGO_URI = "mongodb+srv://{}:{}@{}".format(
+    MONGODB_ATLAS_USERNAME,
+    MONGODB_ATLAS_PASSWORD,
+    MONGODB_ATLAS_CLUSTER_ADDRESS)
+# print("Pymongo URI: {}".format(PYMONGO_URI))
+client = pymongo.MongoClient(PYMONGO_URI)
+testDB = client.test
+eventsDB = client.eventsDB
+events_coll = eventsDB.all_events
+
+
+@app.route('/v1/add', methods=['POST'])
+def add_event():
+    # TODO(cmei4444): Get event information from request data
+    info = request.form['info']
+    # Add information into database
```

Comment:
These sorts of comments can be confusing, since the following 9 lines don't actually add the information to the database if I read it correctly. The actual add happens in `event.add_to_db`, I presume.

Reply:
Took out the comment, since it's clear where the add to db is happening.